power_grid_model_c/power_grid_model/include/power_grid_model/calculation_parameters.hpp

```diff
@@ -108,6 +108,20 @@ template <symmetry_tag sym_type> struct PowerSensorCalcParam {
     RealValue<sym> q_variance{}; // variance (sigma^2) of the error range of the reactive power, in p.u.
 };
 
+// current sensor calculation parameters for state estimation
+// The value is the complex current
+//   * for appliances, it is always in injection direction
+//   * for branches, the direction is node -> branch
+template <symmetry_tag sym_type> struct CurrentSensorCalcParam {
+    using sym = sym_type;
+
+    static constexpr bool symmetric{is_symmetric_v<sym>};
+
+    ComplexValue<sym> value{};
```

Comment:
Angle measurement type should be part of the calculation parameter. We need this to handle differently in the mathematical core.

Reply:
done